packages/scandipwa/src/query/Config.query.js

```diff
@@ -28,7 +28,8 @@ export class ConfigQuery {
             .addFieldList([
                 'id',
                 'label',
-                'value'
+                'value',
+                new Field('exchange_rates').addFieldList(['currency_to', 'rate'])
```

Comment:
Please put the `addFieldList` contents to the new method.
So when someone will want to override these fields, it will override the smallest method possible.
Take a look at the `getCheckoutAgreements` implementation.